### PR TITLE
Moved the custom text app secrets from azure function into config fil…

### PR DESCRIPTION
**Pull request by ZulaMostafa**
based on the issue [#68](https://github.com/microsoft/CogSLanguageUtilities/issues/68), Moved custom text app secrets from azure function to the config file beside CLI indexing tool. And the secrets are being passed as headers using customSkillSets

### CODE REVIEW

File: CustomText.Utilities.CognitiveSearch/Solution/CustomTextSkillsetAzureFunction/Properties/ServiceDependencies/ZulaCogSFunction - Zip Deploy/profile.arm.json [hunk 1]
```diff
@@ -0,0 +1,175 @@
+{
```
Comment (mohamedshabanofficial) [inline CustomText.Utilities.CognitiveSearch/Solution/CustomTextSkillsetAzureFunction/Properties/ServiceDependencies/ZulaCogSFunction - Zip Deploy/profile.arm.json:1]: delete this file!

File: CustomText.Utilities.CognitiveSearch/Solution/CustomTextSkillsetAzureFunction/Program.cs [hunk 2]
```diff
@@ -28,6 +25,9 @@ public static class Program
         {
             log.LogInformation("LUIS-D Extractor and Classifier function");
 
+            var customtextEndpoint = req.Headers["endpointUrl"];
```
Comment (mohamedshabanofficial) [inline CustomText.Utilities.CognitiveSearch/Solution/CustomTextSkillsetAzureFunction/Program.cs:28]: need some checks for these headers!
we need to make sure the fields are not empty and valid
the azure function should be self-sufficient

should include a new error if these secrets aren't working

File: CustomText.Utilities.CognitiveSearch/Solution/CustomTextSkillsetAzureFunction/CustomTextSkillsetAzureFunction.csproj [hunk 3]
```diff
@@ -9,7 +9,7 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="CustomText.MiniSDK">
-      <HintPath>..\..\Release\CustomTextMiniSDKClient.dll</HintPath>
+      <HintPath>..\CustomTextMiniSDK\CustomText.MiniSDK.Client\bin\Debug\netcoreapp3.1\CustomText.MiniSDK.dll</HintPath>
```
Comment (mohamedshabanofficial) [inline CustomText.Utilities.CognitiveSearch/Solution/CustomTextSkillsetAzureFunction/CustomTextSkillsetAzureFunction.csproj:12]: why did you change this?

File: CustomText.Utilities.CognitiveSearch/Solution/CustomTextAppIndexer/CustomTextAppIndexer.ViewLayer.Cli/Properties/launchSettings.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "CognitiveSearchIntegration.ViewLayer.Cli": {
       "commandName": "Project",
-      "commandLineArgs": "index --schema \"C:\\Users\\v-moshaban\\Desktop\\CLutils Workspace\\configs and secrets\\cognitive-search\\CustomTextSchema.json\" --index-name \"devtest\""
+      "commandLineArgs": "index --schema \"CustomTextSchema.json\" --index-name \"devtest2\""
```
Comment (mohamedshabanofficial) [inline CustomText.Utilities.CognitiveSearch/Solution/CustomTextAppIndexer/CustomTextAppIndexer.ViewLayer.Cli/Properties/launchSettings.json:5]: ignore this file also

File: CustomText.Utilities.CognitiveSearch/Solution/CustomTextSkillsetAzureFunction/CustomTextSkillsetAzureFunction.csproj (hunk 3 above)
Comment (ZulaMostafa) [reply CustomText.Utilities.CognitiveSearch/Solution/CustomTextSkillsetAzureFunction/CustomTextSkillsetAzureFunction.csproj:12]: It was missing from the dependencies

File: CustomText.Utilities.CognitiveSearch/Solution/CustomTextAppIndexer/CustomTextAppIndexer.ViewLayer.Cli/Configs/ConfigModels/customtestAppConfigModel.cs [hunk 5]
```diff
@@ -0,0 +1,16 @@
+﻿using Newtonsoft.Json;
```
Comment (mohamedshabanofficial) [inline CustomText.Utilities.CognitiveSearch/Solution/CustomTextAppIndexer/CustomTextAppIndexer.ViewLayer.Cli/Configs/ConfigModels/customtestAppConfigModel.cs:1]: what's CustomTest?!
rename class and file

File: CustomText.Utilities.CognitiveSearch/Solution/CustomTextAppIndexer/CustomTextAppIndexer.Core/Services/CognitiveSearch/CustomTextIndexingService.cs [hunk 6]
```diff
@@ -9,6 +9,13 @@ namespace Microsoft.Azure.CognitiveServices.Language.CustomText.Utilities.Search
 {
     public class CustomTextIndexingService : ICustomTextIndexingService
     {
+        public string customTextEndpoint, customTextKey, customTextModelID;
+        public CustomTextIndexingService(string _customTextEndpoint, string _customTextKey, string _customTextModelID)
```
Comment (mohamedshabanofficial) [inline CustomText.Utilities.CognitiveSearch/Solution/CustomTextAppIndexer/CustomTextAppIndexer.Core/Services/CognitiveSearch/CustomTextIndexingService.cs:13]: .net convention, arguments to functions or constructors should start with lowercase
_ underscore indicates a private class variable
i.e. remove the underscores from parameters
and add underscores to class attributes

File: CustomText.Utilities.CognitiveSearch/Solution/CustomTextSkillsetAzureFunction/CustomTextSkillsetAzureFunction.csproj (hunk 3 above)
Comment (mohamedshabanofficial) [reply CustomText.Utilities.CognitiveSearch/Solution/CustomTextSkillsetAzureFunction/CustomTextSkillsetAzureFunction.csproj:12]: i specifically recall pushing this .dll file
*.dll is ignored by default (as specified in the .gitignore file) 

can you please place this file in the "release" folder and update your reference?
